src/packages/health-check/dashboard-health-check.element.ts

```diff
@@ -58,7 +59,7 @@ export class UmbDashboardHealthCheckElement extends UmbLitElement {
 				weight: 500,
 				meta: {
 					label: group.name || '',
-					api: () => import('./health-check.context.js'),
+					api: UmbHealthCheckContext,
```

Comment:
In order to stay aligned with other manifests, could we then move the `api` property out of the "meta" object and onto the manifest itself?

Reply:
Yeah last thing that I forgot to add 🚀